stac_fastapi/pgstac/stac_fastapi/pgstac/core.py

```diff
@@ -129,7 +129,7 @@ async def _search_base(
         collection = ItemCollection(**items)
         cleaned_features: List[Item] = []
 
-        for feature in collection["features"]:
+        for feature in collection.get("features") or []:
```

Comment:
you can't do `.get('features', [])`? Most `.get` operators in Python allow for a second fallback argument, which just defaults to None

Reply:
This doesn't just account for the case where there is no `features` key, but also where there is a `features` key and the value is `None`